### PR TITLE
Sc 9437 add jenkins doc

### PR DESCRIPTION
**Pull request by wawanawna**

### CODE REVIEW

File: docs/integration/jenkins.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+**jenkins.nodes.builds** <br>*(long counter)*      |  The count of build nodes available to Jenkins
+**jenkins.nodes.builds.time** <br>*(long counter)* *(ms)*      |  The time nodes spend for building
+**jenkins.nodes.offline** <br>*(long gauge)*      |  The count of build nodes available to Jenkins but currently off-line
+**jenkins.nodes.online** <br>*(long gauge)*      |  The count of build nodes available to Jenkins and currently on-line
+**jenkins.queue.blocked** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.buildable** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.pending** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
+**jenkins.queue.size** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue
+**jenkins.queue.stuck** <br>*(long gauge)*      |  The count of jobs that are in the Jenkins build queue and currently in the blocked state
```
Comment (adnanrahic) [inline docs/integration/jenkins.md:36]: This seems wrong...

File: docs/integration/jenkins.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+**jenkins.plugins.inactive** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that are not currently enabled
+**jenkins.plugins.failed** <br>*(long gauge)*      |  The count of plugins in the Jenkins instance that failed to start. A value other than 0 is typically indicative of a potential issue within the Jenkins installation that will either be solved by explicitly disabling the plugin(s) or by resolving the plugin dependency issues
+**jenkins.executors.free** <br>*(long gauge)*      |  The count of executors available to Jenkins that are not currently in use
+**jenkins.executors.in_use** <br>*(long gauge)*      |  The count of executors available to Jenkins that are currently in use
+**jenkins.runs.success** <br>*(long counter)*      |  The count of job runs which performed succeed
```
Comment (adnanrahic) [inline docs/integration/jenkins.md:43]: The count of job runs which performed succeed
=>
The count of job runs which performed successfully

File: docs/integration/jenkins.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+**jenkins.runs.unstable** <br>*(long counter)*      |  The count of job runs which were unstable
+**jenkins.runs.failure** <br>*(long counter)*      |  The count of job runs which failed
+**jenkins.runs.not_built** <br>*(long counter)*      |  The count of not built job runs
```
Comment (adnanrahic) [inline docs/integration/jenkins.md:46]: The count of not built job runs
=>
The count of job runs that were not built

File: docs/integration/jenkins.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+**jenkins.jobs** <br>*(long counter)*      |  The total count of jobs
+**jenkins.jobs.scheduled** <br>*(long counter)*      |  The count at which jobs are scheduled. If a job is already in the queue and an identical request for scheduling the job is received then Jenkins will coalesce the two requests. This metric gives a reasonably pure measure of the load requirements of the Jenkins master as it is unaffected by the count of executors available to the system
+**jenkins.jobs.queuing** <br>*(long counter)*      |  The count of queued jobs
+**jenkins.jobs.blocked** <br>*(long counter)*      |  The count at which jobs in the build queue enter the blocked state
+**jenkins.jobs.buildable** <br>*(long counter)*      |  The count at which jobs in the build queue enter the buildable state
+**jenkins.jobs.execution.time** <br>*(long counter)* *(ms)*      |  The amount of time jobs spend in execution state
+**jenkins.jobs.queuing.time** <br>*(long counter)* *(ms)*      |  The total time jobs spend in the build queue
+**jenkins.jobs.blocked.time** <br>*(long counter)* *(ms)*      |  The amount of time jobs in the build queue enter spend in blocked state
```
Comment (adnanrahic) [inline docs/integration/jenkins.md:55]: The amount of time jobs in the build queue enter spend in blocked state
=>
The amount of time jobs in the build queue spend in blocked state

File: docs/integration/jenkins.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+**jenkins.jobs.buildable.time** <br>*(long counter)* *(ms)*      |  The amount of time jobs in the build queue enter spend in buildable state
```
Comment (adnanrahic) [inline docs/integration/jenkins.md:56]: The amount of time jobs in the build queue enter spend in buildable state
=>
The amount of time jobs in the build queue spend in buildable state

File: docs/integration/jenkins.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+**jenkins.jobs.waiting.time** <br>*(long counter)* *(ms)*      |  The total amount of time that jobs spend in their quiet period
+**jenkins.jobs.total.time** <br>*(long counter)* *(ms)*      |  The time jobs spend from entering the build queue to completing building
```
Comment (adnanrahic) [inline docs/integration/jenkins.md:58]: The time jobs spend from entering the build queue to completing building
=>
The time jobs spend from entering the build queue to completing build

**Review by adnanrahic — CHANGES_REQUESTED** [review]
Some tiny edits.

File: docs/integration/jenkins.md (hunk 1 above)
Comment (wawanawna) [reply docs/integration/jenkins.md]: Sharp eyes, copied from this https://plugins.jenkins.io/metrics/ seems they has wrond values too ))
Changed.

File: docs/integration/jenkins.md (hunk 2 above)
Comment (wawanawna) [reply docs/integration/jenkins.md:43]: done

File: docs/integration/jenkins.md (hunk 3 above)
Comment (wawanawna) [reply docs/integration/jenkins.md:46]: done

File: docs/integration/jenkins.md (hunk 4 above)
Comment (wawanawna) [reply docs/integration/jenkins.md:55]: done

File: docs/integration/jenkins.md (hunk 5 above)
Comment (wawanawna) [reply docs/integration/jenkins.md:56]: done

File: docs/integration/jenkins.md (hunk 6 above)
Comment (wawanawna) [reply docs/integration/jenkins.md:58]: done

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**